resource_alks_iamrole.go

```diff
@@ -167,13 +170,10 @@ func resourceAlksIamTrustRoleCreate(d *schema.ResourceData, meta interface{}) er
 	response := *resp
 
 	d.SetId(response.RoleName)
-	d.Set("arn", response.RoleArn)
-	d.Set("ip_arn", response.RoleIPArn)
-	d.Set("role_added_to_ip", response.RoleAddedToIP)
```

Comment:
I think this value is _only_ available when the role is created and I believe we've historically stuff that in the TF state.

Reply:
Good catch @webbbarker , fixed that. Re-added it to both create IAM Trust Role and regular Role. 👍🏼